packages/profile-sync-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.3]
+
+### Uncategorized
```

Comment:
If possible, try providing a category (added, changed, updated, etc). Feel free to look at other changelogs.